homework_1/nucl_acids.py

```diff
@@ -0,0 +1,96 @@
+dna_to_rna = {'A': 'U', 'a': 'u', 'T': 'A', 't': 'a', 'G': 'C', 'g': 'c', 'C': 'G', 'c': 'g'}
+rna_to_dna = {'A': 'T', 'a': 't', 'u': 'A', 'U': 'a', 'G': 'C', 'g': 'c', 'C': 'G', 'c': 'g'}
+dna_to_dna = {'A': 'T', 'a': 't', 'T': 'A', 't': 'a', 'G': 'C', 'g': 'c', 'C': 'G', 'c': 'g'}
+rna_to_rna = {'A': 'U', 'a': 'u', 'U': 'A', 'u': 'a', 'G': 'C', 'g': 'c', 'C': 'G', 'c': 'g'}
+
+
+def check_sequence(seq):
+    seq_check = seq.upper()
+
+    if seq_check.count('A') + seq_check.count('T') + seq_check.count('G') + seq_check.count('C') == len(seq_check):
```

Comment:
Очень неэффективный способ, `str.count` внутри себя проходит по всей строке, т.е. суммарно получаем 4 прохода. Мы не проходили, но можно это очень удобно сделать при помощи множеств.
```suggestion
    if set(seq_check) <= {"A", "T", "G", "C"}:
```
<= - оператор проверки на одмножество